pdns/libssl.hh

```diff
@@ -100,6 +100,7 @@ public:
 #if OPENSSL_VERSION_MAJOR >= 3
   int encrypt(unsigned char keyName[TLS_TICKETS_KEY_NAME_SIZE], unsigned char* iv, EVP_CIPHER_CTX* ectx, EVP_MAC_CTX* hctx) const;
   bool decrypt(const unsigned char* iv, EVP_CIPHER_CTX* ectx, EVP_MAC_CTX* hctx) const;
+  [[nodiscard]] std::string content() const;
```

Comment:
This line should be outside the `#if` guard

Reply:
Ugh, thanks for noticing it and sorry I'm missed it _twice_! I pushed a backport of your commit from https://github.com/PowerDNS/pdns/pull/14419